annotation/src/main/java/io/smallrye/common/annotation/SuppressForbidden.java

```diff
@@ -0,0 +1,15 @@
+package io.smallrye.common.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used to suppress forbidden-apis errors inside the element it's added to.
+ */
+@Retention(RetentionPolicy.CLASS)
+@Target({ ElementType.CONSTRUCTOR, ElementType.FIELD, ElementType.METHOD, ElementType.TYPE })
+public @interface SuppressForbidden {
+    String reason();
```

Comment:
```suggestion
	@AliasFor("reason")
	String[] value() default {};

	@AliasFor("value")
	String[] reason() default {};
```
might overload like done in org.springframework.context.annotation.Bean.

Reply:
If possible I would prefer to maintain `reason` just for us to be able to change it in the future if needed.

Reply:
> bad idea

the idea is not enforcing some **random param**, as the topic is kind clear, you just mention it, there wont be any more very likely.

Considering demo PR, its about avoiding **boilerplate** and go with concise API, just like spring does.
Providing the param for devs who like more overhead and verbose code can do so, just like in spring. 

- https://github.com/quarkusio/quarkus/pull/48167
- https://www.baeldung.com/spring-aliasfor-annotation#aliasfor

As the property seems not used anyway its pure sugar.